src/literature/components/ReferenceItem.jsx

```diff
@@ -1,51 +1,59 @@
 import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { Map } from 'immutable';
-import { List } from 'antd';
+import { List, Row, Col } from 'antd';
 import { Link } from 'react-router-dom';
 
 import AuthorList from '../../common/components/AuthorList';
 import Latex from '../../common/components/Latex';
 import PublicationInfoList from '../../common/components/PublicationInfoList';
 
 class ReferenceItem extends Component {
+  static renderLabel(reference) {
+    const label = reference.get('label');
+    const labelDisplay = label ? `[${label}] ` : null;
+    return labelDisplay;
+  }
+
   static renderTitle(reference) {
     const recordId = reference.get('control_number');
     const title = reference.getIn(['titles', 0, 'title'], '');
-    const label = reference.get('label');
-
-    const labelDisplay = label ? `[${label}] ` : '';
 
     if (recordId && title) {
       return (
-        <Link to={`/literature/${recordId}`}>
-          {labelDisplay} <Latex>{title}</Latex>
+        <Link to={`/literature/${recordId}`} className="f5">
```

Comment:
we don't need this here :)

Reply:
I mean the ``className``

Reply:
This is to make the title appear bigger if it is linked to a record in inspire

Reply:
But this will be solved in a different commit

Reply:
True. I'll undo this